be-scala-akka/Jenkinsfile.template

```diff
@@ -29,6 +29,14 @@ def stageBuild(def context) {
   stage('Build and Unit Test') {
     withEnv(["TAGVERSION=${context.tagversion}"]){
       sh "sbt test"
+
+      sh 'pwd'
+      sh 'ls -la target/test-reports'
+      junit 'target/test-reports/**/*.xml'
+      sh 'mkdir -p build/test-results/test'
+      sh 'cp target/test-reports/*.xml build/test-results/test/'
+      sh 'ls -la build/test-results/test'
+
```

Comment:
@stitakis please cleanup the code of `pwd`and `ls` commands. If you want verbosity you could add the respective flag in the `cp` command.

Reply:
Why dont you use test Location??